File: src/_1ProgrammingBasicsWithJavaMay2023/_03ConditionalStatementsAdvanced/_1Lab/_09FruitOrVegetable.java
package _1ProgrammingBasicsWithJavaMay2023._03ConditionalStatementsAdvanced._1Lab;

import java.util.Scanner;

public class _09FruitOrVegetable {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine();
        switch (input) {
            case "banana":
            case "apple":
            case "kiwi":
            case "cherry":
            case "lemon":
            case "grapes":
                System.out.println("fruit");
                break;
            case "tomato":
            case "cucumber":
            case "pepper":
            case "carrot":
                System.out.println("vegetable");
                break;
            default:
                System.out.println("unknown");
                break;
        }
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_3ConditionalStatementsAdvanced/_1Lab/_09FruitOrVegetable.js
function fruitOrVegetables(product) {
    let output = "";
    if (product === "banana" || product === "apple" || product === "kiwi" || product === "cherry" || product === "lemon" || product === "grapes") {
        output = "fruit";
    } else if (product === "tomato" || product === "cucumber" || product === "pepper" || product === "carrot") {
        output = "vegetable";
    } else {
        output = "unknown";
    }

    console.log(`${output}`)
}